db/schema/cachedClientTools.ts

```
@@ -0,0 +1,29 @@
+import { bigint, index, jsonb, pgTable, sql, text, uniqueIndex } from "drizzle-orm/pg-core";
```

ESLint

[error] 'index' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)